src/serviceorchestration/5_0_0/sd/orchestration/orchestration_sd.tex

```diff
@@ -0,0 +1,441 @@
+\rowcolor{gray!33} Field & Type & Mandatory & Description \\ \hline
+serviceDefinition & \pref{Name} & yes/no & The required service definition name. Mandatory in case of \textbf{dynamic} strategy. \\ \hline
+operations & \pref{List}$<$\pref{Name}$>$ & yes/no & The required service operation names. \newline 
+Exactly one operation must be defined, when the following orchestration flags are true: \newline \texttt{ONLY\_INTERCLOUD}, \texttt{ALLOW\_INTERCLOUD}, \texttt{ALLOW\_TRANSLATION} \\ \hline
+versions & \pref{List}$<$\pref{Version}$>$ & no & The required service versions. \\ \hline 
+alivesAt & \pref{DateTime} & no & The orchestrated service must be alive by this time. \\ \hline
+metadataRequirements & \pref{List}$<$\hyperref[sec:model:MetadataRequirements]{MetadataRequirements}$>$ & no & The orchestrated service must meet at least to one of the specified metadate reuirement. \\ \hline
+interfaceTemplateNames & \pref{List}$<$\pref{InterfaceTemplate}$>$ & no & The orchestrated service must offer at least one from the specified interface template names. \\ \hline 
+interfaceAddressTypes & \pref{List}$<$\pref{AddressType}$>$ & no & The orchestrated service must offer at least one from the specified interface address types. \\ \hline
+interfacePropertyRequirements & \pref{List}$<$\hyperref[sec:model:MetadataRequirements]{MetadataRequirements}$>$ & no & The orchestrated service must offer at least one interface that meets with one of the specified property requirements. \\ \hline
+securityPolicies & \pref{List}$<$\pref{SecurityPolicy}$>$ & no & The orchestrated service must meet with one of the specified security policies. \\ \hline
+preferredProviders & \pref{List}$<$\pref{Name}$>$ & no & Provider system names specified here have priority. \\ \hline
+\end{tabularx}
+\end{table}
+
+\msubsection{struct}{MetadataRequirements}
+
+A special \pref{Object} which maps \pref{String} keys to \pref{Object}, primitive or list values, where 
+
+\begin{itemize}
+    \item Keys can be paths (or multi-level keys) which access a specific value in a \hyperref[sec:model:Metadata]{Metadata} structure, where parts of the path are delimited with dot character (e.g. in case of "key.subkey" path we are looking for the key named "key" in the metadata, which is associated with an embedded object and in this object we are looking for the key named "subkey").
+    \item Values are special \pref{Object}s with two fields: an operation (e.g. less than) and an actual value (e.g. a number). A metadata is matching a requirement if the specified operation returns true using the metadata value referenced by a key path as first and the actual value as second operands. 
+    \item Alternatively, values can be ordinary primitives, lists or \pref{Object}s. In this case the operation is equals by default.
+\end{itemize}
+
+\msubsection{struct}{OrchestrationFlag}
+
+Specific \pdef{String}:\pdef{Boolean} pair to control the orchestration process. Possible values:
```

Comment:
Somewhere here we should mentioned that not all strategies are using all orchestration flags. And maybe for each flag we should mention which strategy will use it (in the design Excel sheet, we specify this if I remember right)